arch/s390/s390_features.c

```diff
@@ -1,10 +1,14 @@
 #include "../../zbuild.h"
 #include "s390_features.h"
 
-#include <sys/auxv.h>
+#if defined(HAVE_SYS_AUXV_H)
+#  include <sys/auxv.h>
+#endif
 
-Z_INTERNAL int s390_cpu_has_vx;
+Z_INTERNAL int s390_cpu_has_vx = 0;
 
 void Z_INTERNAL s390_check_features(void) {
+#if defined(S390_FEATURES)
```

Comment:
Should just use `ifdef`?

Reply:
Both works if there is only one test on a line... I can change that...